modules/monitoring-attaching-additional-labels-to-your-time-series-and-alerts.adoc

```diff
@@ -135,8 +135,3 @@ data:
 ----
 
 . Save the file to apply the changes. The new configuration is applied automatically.
```

Comment:
@juzhao  When I tested this, this change did not trigger pods to redeploy so I am not including the sentence about the redeployment here.

Reply:
correct, no pods redeployed, the change is updated to prometheus configuration
```
$ oc -n openshift-monitoring exec -c prometheus prometheus-k8s-0 -- cat /etc/prometheus/config_out/prometheus.env.yaml | head
global:
  evaluation_interval: 30s
  scrape_interval: 30s
  external_labels:
    environment: prod
    prometheus: openshift-monitoring/k8s
    prometheus_replica: prometheus-k8s-0
    region: eu
```